test/lib/widgets/product_license_confirmation_test.rb

```diff
@@ -125,7 +125,8 @@
       context "and license is still unconfirmed" do
         let(:license_confirmed?) { false }
 
-        it "returns false" do
+        it "returns false and Report it" do
```

Comment:
NP: "reports the error and returns false"

Reply:
will do thanks.